apps/webpack.js

```diff
@@ -99,7 +99,7 @@ var storybookConfig = _.extend({}, baseConfig, {
 
 // config for our test runner
 var karmaConfig = _.extend({}, baseConfig, {
-  devtool: 'cheap-module-source-map',
+  devtool: 'inline-source-map',
```

Comment:
not strictly necessary, but makes it so that i can actually debug when i do WATCH=1 BROWSER=Chrome (means we have same devtool as storybook).